### DISCUSSION TITLE
Did it succeed in flashing the esc?

### DISCUSSION THREAD

**Discussion by Mysterious-Wrangler** [timeylies/How-to-convert-a-Ninebot-SNSC-2.2-or-2.3-Scooter#2] [Issues]
Hi, i am on the flashing part for ninebot snsc 2.2 esc and its a bit janky, at times it connects to target and other times it doesnt connect. Anyways i somehow got it connected and apparently flashed the esc but after that it disconnects every time. Did i succeed flashing the esc? I also tried linking 5v from stlink to the pin in your picture with the same results(i didnt touch any settings other than disable readout protection to make it work. I attached a picture. Thanks.
![ac](https://github.com/user-attachments/assets/cc9faacb-b667-46a7-a813-998db96e1738)

**Comment by timeylies** [comment]
Yeah it succeeded, it does that for me too
